pkg/tx_list_validator/tx_list_validator.go

```diff
@@ -19,24 +19,21 @@ const (
 )
 
 type TxListValidator struct {
-	blockMaxGasLimit        uint64
-	maxTransactionsPerBlock uint64
-	maxBytesPerTxList       uint64
-	chainID                 *big.Int
+	blockMaxGasLimit  uint64
+	maxBytesPerTxList uint64
+	chainID           *big.Int
 }
 
 // NewTxListValidator creates a new TxListValidator instance based on giving configurations.
 func NewTxListValidator(
 	blockMaxGasLimit uint64,
-	maxTransactionsPerBlock uint64,
```

Comment:
According to Brecht's comment here: https://www.notion.so/taikoxyz/Pseudocode-build_l2_block-d2004fa069184c1fad013e4113e25cab , seems we should add this param back, this config was removed from protocol, but need to be hard-coded in circuits and nodes.

Reply:
Ah I see, I will get the number from Brecht and re add